client/extensions/woocommerce/state/products/selectors.js

```diff
@@ -0,0 +1,13 @@
+
+/**
+ * Gets product fetched from server.
+ *
+ * @param {Object} state Global state tree
+ * @param {Number} productId Numeric product Id.
+ * @return {Object} Product object from API.
+ */
+export function getProduct( state, productId ) {
```

Comment:
I think you might be right. But since this is just a placeholder, I think we can worry about that in a future PR.